infrastructure/bin/atat-web-api.ts

```diff
@@ -7,7 +7,9 @@ import { getTags } from "../lib/load-tags";
 const app = new cdk.App();
 const accountId = app.node.tryGetContext("account");
 const region = app.node.tryGetContext("region");
-const stack = new AtatWebApiStack(app, "AtatWebApiStack", {
+// Ugly hack to quickly isolate deployments for developers.  To be improved/removed later.
+const ticketId = app.node.tryGetContext("TicketId") || "";
```

Comment:
```suggestion
const ticketId = app.node.tryGetContext("TicketId") || "";
if (!ticketId) {
  console.warn("NOTE: No ticket ID has been provided. Please be sure this is the intended behavior. You may overwrite someone's stack");
}
```

maybe something like this

Reply:
I've added something like this in later commit.